app/grants/models.py

```diff
@@ -1173,7 +1173,8 @@ def create_contribution(self, tx_id):
         self.save()
         grant.updateActiveSubscriptions()
         grant.save()
-        successful_contribution(self.grant, self, contribution)
```

Comment:
@mds1 / @octavioamu this is what always causes the mail to be sent asap !
For cross chain flows it will happen only after txn is validated 
For eth -> we will have to shift this to the celery task which does the validation 

`process_grant_contribution`  can do this in follow up PR after running it with team